modules/develop/pages/data-transforms/build.adoc

```diff
@@ -46,11 +67,15 @@ curl -H "Authorization: Bearer ${RP_CLOUD_TOKEN}" -X PATCH \
  -d '{"cluster_configuration":{"custom_properties": {"data_transforms_enabled":true}}}'
 ----
 
-The xref:api:ROOT:cloud-controlplane-api.adoc#patch-/v1/clusters/-cluster.id-[`PATCH /clusters/{cluster.id}`] request returns the ID of a long-running operation. The operation may take up to ten minutes to complete. You can check the status of the operation by polling the xref:api:ROOT:cloud-controlplane-api.adoc#get-/v1/operations/-id-[`GET /operations/\{id}`] endpoint.
+NOTE: This cluster configuration operation may take up to ten minutes to complete. The xref:api:ROOT:cloud-controlplane-api.adoc#patch-/v1/clusters/-cluster.id-[`PATCH /clusters/{cluster.id}`] request returns the operation ID. You can check the status of the operation by polling the xref:api:ROOT:cloud-controlplane-api.adoc#get-/v1/operations/-id-[`GET /operations/\{id}`] endpoint. 
```

Comment:
Let's add the similar note here 'Some cluster configuration changes will result in a rolling restart.